### PR TITLE
PEPPER-37  Add new auth0-log-event-code

### PR DESCRIPTION
**Pull request by ssettipalli**
look at https://broadworkbench.atlassian.net/browse/PEPPER-37
Add new auth0 log event code:

### CODE REVIEW

File: pepper-apis/dss-database/src/main/resources/db-changes/seed/PEPPER-37-add-auth0-log-event-code.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet author="sampath" id="20221013-add-auth0-log-event-code-table">
+        <sql>
+        insert into auth0_log_event_code (code, title, description)
+        values ('signup_pwd_leak', 'Breached password', 'Someone behind the IP address: ip attempted to login with a leaked password.');
```
Comment (bskinner) [inline pepper-apis/dss-database/src/main/resources/db-changes/seed/PEPPER-37-add-auth0-log-event-code.xml]: The `description` reads awkwardly. "Someone behind the IP address" - what IP address? Would a description such as "A login was attempted with a leaked password", and removing the IP language entirely suffice?

Comment (ssettipalli) [reply pepper-apis/dss-database/src/main/resources/db-changes/seed/PEPPER-37-add-auth0-log-event-code.xml]: @bskinner : I believe we tried to replicate what comes from Auth0
https://broadinstitute.slack.com/archives/GCPRWSSCW/p1665521920093989

Comment (bskinner) [reply pepper-apis/dss-database/src/main/resources/db-changes/seed/PEPPER-37-add-auth0-log-event-code.xml]: Understandable, but that description makes little sense without having the IP address it's referring to.

Comment (ssettipalli) [reply pepper-apis/dss-database/src/main/resources/db-changes/seed/PEPPER-37-add-auth0-log-event-code.xml]: updated

**Review by bskinner — APPROVED** [review]
Thank you!

**Pull request merged**